litmus-portal/frontend/src/views/Home/RecentActivity/RecentActivityListItem.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {
+  ListItem,
+  ListItemAvatar,
+  ListItemText,
+  Typography,
+} from '@material-ui/core';
+import { Message } from '../../../models/header';
+
+import useStyles from './style';
+
+interface RecentActivityListItemProps {
+  message: Message;
+}
+
+const RecentActivityListItem: React.FC<RecentActivityListItemProps> = ({
+  message,
+}) => {
+  const classes = useStyles();
+  return (
+    <ListItem>
+      <ListItemAvatar>
+        <Typography className={classes.messageID}>
+          {' '}
+          &nbsp; &nbsp; {message.id}
```

Comment:
please do not hardcode spacing into html

Reply:
Changing it..